docs/usertutorial.rst

```diff
@@ -0,0 +1,455 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Exploring data with Apache Superset
+===================================
+
+In this tutorial, we will introduce key concepts in Apache Superset through
+the exploration of a real dataset which contains the flights made by employees
+of a UK-based organization in 2011. The following information about each
+flight is given:
+
+- The traveller's department. For the purposes of this tutorial
+  the departments have been renamed Orange, Yellow and Purple.
+- The cost of the ticket.
+- The travel class (Economy, Premium Economy, Business and First Class).
+- Whether the ticket was a single or return.
+- The date of travel.
+- Information about the origin and destination.
+- The distance between the origin and destination, in kilometers (km).
+
+Obtaining and loading the data
+------------------------------
+
+Download the data for this tutorial to your computer from `Github
+<https://raw.githubusercontent.com/apache-superset/examples-data/master/tutorial_flights.csv>`_.
+
+In the top menu, select **Sources > Upload a CSV**.
```

Comment:
By default no databases are allowed for CSV upload. Add a step to show how to change the examples DB connection or create the `upload_csv` db connection

Reply:
I've added a section explaining how to change the examples database connection